beacon-chain/rpc/validator/assignments.go

```diff
@@ -155,7 +155,6 @@ func (vs *Server) duties(ctx context.Context, req *ethpb.DutiesRequest) (*ethpb.
 
 			nextAssignment.ValidatorIndex = idx
 			nextAssignment.Status = assignmentStatus(s, idx)
```

Comment:
Can you add a comment here why nextAssignment doesn't have proposer slots?

Reply:
Done